pkgs/top-level/aliases.nix

```diff
@@ -441,6 +441,7 @@ mapAliases {
   copilot-language-server-fhs = lib.warnOnInstantiate "The package set `copilot-language-server-fhs` has been renamed to `copilot-language-server`." copilot-language-server; # Added 2025-09-07
   copper = throw "'copper' has been removed, as it was broken since 22.11"; # Added 2025-08-22
   cordless = throw "'cordless' has been removed due to being archived upstream. Consider using 'discordo' instead."; # Added 2025-06-07
+  corepack_latest = throw "'corepack_latest' has been removed, use 'corepack.overrideAttrs { nodejs = pkgs.nodejs_latest; }' instead"; # Added 2025-10-25
```

Comment:
Out of curiosity, do you have a reference where upstream documents that removal, e.g. a changelog entry?